generators/app/prompts.js

```diff
@@ -380,3 +427,17 @@ function pushComponentsByEntity(generator, answers, entity) {
         generator.props.componentsByEntityConfig[entity] = [...answers.currentEntityComponents];
     }
 }
+
+function pushTopicName(generator, answers) {
+    const name = generator.props.currentEntity === constants.NO_ENTITY ? generator.props.currentPrefix : generator.props.currentEntity;
+
+    if (answers.topic) {
+        if (answers.topic === constants.CUSTOM_TOPIC) {
+            generator.props.topics.push({ key: name, value: answers.topicName });
+        } else if (answers.topic === constants.DEFAULT_TOPIC) {
+            generator.props.topics.push({ key: name, value: utils.topicNamingFormat(generator, name) });
```

Comment:
when i use the incremental mode with the default topic choice, it generates a wrong topic name. (undefined instead of appName in snakeCase)
![WrongTopicName](https://user-images.githubusercontent.com/44546269/84128903-8342a300-aa41-11ea-8829-8c8a0cee3ec1.png)